Sources/AppStoreConnectCLI/Model/BetaGroup.swift

```diff
@@ -5,7 +5,8 @@ import Combine
 import Foundation
 import SwiftyTextTable
 
-struct BetaGroup: TableInfoProvider, ResultRenderable, Equatable {
+struct BetaGroup: TableInfoProvider, ResultRenderable {
+    let id: String?
```

Comment:
Same goes for BetaGroup

Reply:
An optional id is for creating a beta group. When creating a new group we don't have the id yet. we can basically create a file with just minimal information (eg. group name, link enabled, etc) and push back